FSBeheer/FSBeheer/ViewModel/EditQuestionnaireViewModel.cs

```diff
@@ -1,19 +1,105 @@
-﻿using FSBeheer.VM;
-using System;
+﻿using FSBeheer.Model;
+using FSBeheer.View;
+using FSBeheer.VM;
+using GalaSoft.MvvmLight.Command;
+using System.Collections;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Windows.Input;
+using System;
+using GalaSoft.MvvmLight;
+using GalaSoft.MvvmLight.Messaging;
 
 namespace FSBeheer.ViewModel
 {
-    public class EditQuestionnaireViewModel
+    public class EditQuestionnaireViewModel : ViewModelBase
     {
-        private QuestionnaireVM _questionnaireVM;
+        private QuestionnaireVM _questionnaire;
+        private CustomFSContext _context;
+        public QuestionnaireVM Questionnaire
+        {
+            get { return _questionnaire; }
+            set
+            {
+                _questionnaire = value;
+                base.RaisePropertyChanged("Questionnaire");
+            }
+        }
+
+        private QuestionVM _selectedQuestion;
+        public QuestionVM SelectedQuestion {
+            get
+            {
+                return _selectedQuestion;
+            }
+            set
+            {
+                _selectedQuestion = value;
+                base.RaisePropertyChanged("SelectedQuestion");
+            }
+        }
+        public ObservableCollection<int?> InspectionNumbers { get; set; }
```

Comment:
<int?> Wat houdt dit in?

Reply:
Nullable int. Het kan helaas geen normale int zijn.